README.md

```diff
@@ -119,6 +119,10 @@ optional features are available:
 - `serde1` enables serialization for some types, via Serde version 1.
 - `stdweb` enables support for `OsRng` on `wasm-unknown-unknown` via `stdweb`
   combined with `cargo-web`.
+- `stdweb` enables support for `OsRng` on `wasm-unknown-unknown` via
```

Comment:
s/stdweb/wasm-bindgen/

Also `wasm-unknown-unknown` should be `wasm32-unknown-unknown` here and the existing spot above.